protobuf/server_message.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package pganalyze.collector;
+option go_package = "github.com/pganalyze/collector/output/pganalyze_collector";
+
+message ServerMessage {
+  oneof message {
+    // Collector configuration
+    Config config = 1;
+    // Server request to pause collection of data (e.g. in case of duplicate collectors)
+    Pause pause = 2;
+    // Request for the collector to generate an explain plan for a query
+    ExplainRun explain_run = 3;
+  }
+
+  message Config {
+    string server_id = 1;
+    string server_url = 2;
```

Comment:
We should check this server URL instead in the part where we output the URL to view the server:

https://github.com/pganalyze/collector/blob/f9752507c41c0fab792d5760beaea17504054372/selftest/summary.go#L222

(and drop server.PGAnalyzeURL)

Reply:
Also when we do that we should make sure that we still see the server URL at the bottom of `--test` for both the HTTP and the WebSocket mechanism.